common/fw/CMakeLists.txt

```diff
@@ -4,7 +4,7 @@ cmake_minimum_required(VERSION 3.1.3)
 
 project(uvc_fw)
 
-file(READ "firmware-version.h" ver)
+file(READ "${CMAKE_CURRENT_LIST_DIR}/firmware-version.h" ver)
```

Comment:
I still think this shouldn't be needed.  Do you know which version of `cmake` you are using?

Reply:
3.10.2. This change is only required when I build from Android Studio